### PR TITLE
use `pyproject.toml` instead of `setup.py`

### PR DESCRIPTION
**Pull request by mbalatsko**
- use `pyproject.toml` instead of `setup.py`
- handle license according to recommendation in https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

Small tweak to modernize package definition :)

### CODE REVIEW

**Comment by di** [comment]
@mbalatsko Thanks! Looks like CI is failing, not sure if that's due to this PR or not (has been a while since it's run).

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "bump"
+version = "1.3.2"
+description = "Bumps package version numbers"
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {file = "LICENSE"}
```
Comment (di) [inline pyproject.toml:11]:
```suggestion
```
See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

**Comment by mbalatsko** [comment]
> @mbalatsko Thanks! Looks like CI is failing, not sure if that's due to this PR or not (has been a while since it's run).

I found an issue `setup.py` does not exist in path now, and CliRunner fails with invalid input, which is correct. I'll fix the test

File: pyproject.toml (hunk 1 above)
Comment (mbalatsko) [reply pyproject.toml:11]: Do you want to get rid of it completely or at least leave as text?

Comment (mbalatsko) [reply pyproject.toml]: Does this look better from your side?

Comment (di) [reply pyproject.toml]: It can be removed entirely, see the last line in that link.

Comment (mbalatsko) [reply pyproject.toml]: Ok, pushed

**Comment by mbalatsko** [comment]
@di is there is something more I should add to consider this PR ready to merge?

**Review by di — APPROVED** [review]

**Comment by di** [comment]
Thanks!

**Pull request merged**